tests/components/vera/test_config_flow.py

```diff
@@ -0,0 +1,108 @@
+"""Vera tests."""
+from unittest.mock import MagicMock
+
+from mock import patch
+from requests.exceptions import RequestException
+
+from homeassistant.components.vera import CONF_CONTROLLER
+from homeassistant.components.vera.config_flow import VeraFlowHandler
+from homeassistant.const import CONF_EXCLUDE, CONF_LIGHTS
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+
+
+async def test_aync_step_user_success(hass: HomeAssistant) -> None:
+    """Test function."""
+    with patch("pyvera.VeraController") as vera_controller_class_mock:
+        controller = MagicMock()
+        controller.refresh_data = MagicMock()
+        vera_controller_class_mock.return_value = controller
+
+        handler = VeraFlowHandler()
+        handler.hass = hass
+        base_url = "http://127.0.0.1/"
+
+        result = await handler.async_step_user()
```

Comment:
Please use the core interfaces to set up the tests. It will make the tests more robust.

```py
result = await hass.config_entries.flow.async_init(
    DOMAIN, context={"source": "user"}
)
```

https://github.com/home-assistant/home-assistant/blob/16199b7a52146c563de36ce5e50cd1b59dbed03c/homeassistant/data_entry_flow.py#L94-L96